src/cloud/paths/orgs_orgID_limits.yml

```diff
@@ -1,27 +1,5 @@
 get:
-  tags:
-    - Limits
-  summary: Get limit for an org
-  parameters:
-    - in: path
-      name: orgID
-      description: org to get limit for
-      required: true
-      schema:
-        type: string
-  responses:
-    '200':
-      description: A limit
-      content:
-        application/json:
-          schema:
-            $ref: "../schemas/Limit.yml"
-    default:
-      description: unexpected error
-      content:
-        application/json:
-          schema:
-            $ref: "../../common/schemas/Error.yml"
+  $ref: "./orgs_orgID_limits_get.yml"
```

Comment:
this works, but seems too granular. was there a reason to externalize the `get` definition

Reply:
Yeah- the existing limits API was only wired under `api/v2private/` and only the GET operation was appropriate for wiring into a public route under `/api/v2/`.

But that said, this reminds me that the response is _slightly_ different, because of [this] piece that causes the public response to be wrapped in the usual wrapper with `links` etc.

So TLDR i think I can just add a new public route and schema for this rather than reusing the private limits bits here.

Reply:
> ya, looks like it was slightly "broken" before (the schema type wasn't defined in the schema portion of the swagger because in the `-diff` swagger we don't import common types), so those changes can be ignored.
> I think iris may want to verify these changes too, since her team consumes this bit as well, if i understand correctly.

I'll request review from Iris when I think this is good to go.